### PR TITLE
NXP-24525: fix file property propagation on nuxeo-document-import

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: elements/document/nuxeo-document-import.html [hunk 1]
```diff
@@ -708,15 +708,22 @@
...
-            // hack: refresh complete on the dom repeat
-            var i = this.localFiles.findIndex(function(f) {
-              return f.index === this.get(changeRecord.path.replace('.index', '')).index;
-            }.bind(this));
-            this.notifyPath(['localFiles', i, 'complete'].join('.'), true);
+          } else {
+            var match = changeRecord.path.match(/(files\.\d+)\.(\w+)/);
+            if (match) {
+              // this is needed because the client doesn't account for removed blobs
+              var file = this.get(match[1])
```
Comment (afjusto) [inline elements/document/nuxeo-document-import.html]: missing a `;`

File: elements/document/nuxeo-document-import.html [hunk 2]
```diff
@@ -708,15 +708,22 @@
...
+              var localFilesIdx = this.localFiles.findIndex(function(f) {
+                return f === file;
```
Comment (afjusto) [inline elements/document/nuxeo-document-import.html]: if you are comparing the whole object, couldn't you use the regular `indexOf`?

**Comment by afjusto** [comment]
👍

File: elements/document/nuxeo-document-import.html (hunk 1 above)
Comment (Gabez0r) [reply elements/document/nuxeo-document-import.html]: 👍

File: elements/document/nuxeo-document-import.html (hunk 2 above)
Comment (Gabez0r) [reply elements/document/nuxeo-document-import.html]: I sure can. Was comparing a property before and just blindly replaced the function body. Thanks. 👍

**Comment by adilioaraujo** [comment]
:+1:

**Pull request merged**